### PR TITLE
Update owners for ContribEx

### PR DESCRIPTION
**Pull request by alisondy**
/hold for review
/assign @mrbobbytables @cblecker @nikhita

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -8,9 +8,10 @@ approvers:
   - mrbobbytables
   - nikhita
   - parispittman
-  - spiffxp
   - committee-steering
 emeritus_approvers:
   - castrojo
   - calebamiles
   - grodrigues3
+  - spiffxp
```
Comment (alisondy) [inline OWNERS:16]: @spiffxp , Would you like to remain as an emeritus approver in the root owners file?

File: communication/OWNERS [hunk 2]
```diff
@@ -2,16 +2,17 @@
...
   - jeefy
   - mrbobbytables
   - markyjackson-taulia
 approvers:
-  - castrojo
+  - mrbobbytables
   - parispittman
   - sig-contributor-experience-leads
+emeritus_approvers:
+  - castrojo
```
Comment (alisondy) [inline communication/OWNERS:15]: @castrojo , Would you like to remain as an emeritus approver in the communication owners file?

File: communication/marketing-team/OWNERS [hunk 3]
```diff
@@ -4,13 +4,13 @@ reviewers:
...
   - parispittman
-  - mbbroberg
 approvers:
   - parispittman
-  - mbbroberg
   - Pensu
   - rajula96reddy
   - kaslin
+emeritus_approvers:
+  - mbbroberg
```
Comment (alisondy) [inline communication/marketing-team/OWNERS:13]: cc @mbbroberg Would you like to remain as an emeritus approver in the marketing team owners file?

File: contributors/guide/OWNERS [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
-  - guineveresaenger
-  - idvoretskyi
-  - tpepper
 approvers:
-  - castrojo
+  - mrbobbytables
   - parispittman
+emeritus_approvers:
+  - castrojo
   - guineveresaenger
```
Comment (alisondy) [inline contributors/guide/OWNERS:8]: @castrojo , @guineveresaenger 

Would you like to remain as emeritus approvers in the contributor guide owners file?

File: contributors/guide/OWNERS [hunk 5]
```diff
@@ -1,13 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-reviewers:
-  - castrojo
-  - guineveresaenger
-  - idvoretskyi
-  - tpepper
```
Comment (alisondy) [inline contributors/guide/OWNERS:7]: @idvoretskyi , @tpepper 

Would you like to remain as reviewers in the contributor guide owners file?

File: events/OWNERS [hunk 6]
```diff
@@ -1,12 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-reviewers:
-  - parispittman
-  - tpepper
```
Comment (alisondy) [inline events/OWNERS:5]: @tpepper 

Would you like to remain as a reviewer in the events owners file?

File: events/OWNERS [hunk 7]
```diff
@@ -1,12 +1,8 @@
...
 approvers:
   - parispittman
   - mrbobbytables
-  - guineveresaenger
```
Comment (alisondy) [inline events/OWNERS:9]: @guineveresaenger 

Would you like to remain as an approver in the events owners file?

File: sig-contributor-experience/devstats/OWNERS [hunk 8]
```diff
@@ -1,13 +1,12 @@
...
-  - dims
-  - nikhita
 approvers:
-  - lukaszgryglicki
+  - dims
   - jberkus
   - LappleApple
+  - mrbobbytables
 emeritus_approvers:
+  - lukaszgryglicki
```
Comment (alisondy) [inline sig-contributor-experience/devstats/OWNERS:9]: @lukaszgryglicki 

Would you like to remain as an emeritus approver in the devstats owners file?

**Review by alisondy — COMMENTED** [review]
I've pinged all the folks who have been removed, or changed status in the review.

We will go forward with this in two weeks, and if there are any issues, we can restore back.

**Comment by jdumars** [comment]
Adding my LGTM so we can merge when the folks involved have resolved their roles moving forward. Thank you for te care that has gone into this process.

/lgtm

**Comment by jdumars** [comment]
Hey folks, just checking in on this since it has been a while. Any blockers preventing this from approval?

**Pull request merged**